### PR TITLE
2.1 Announcement

### PR DESCRIPTION
**Pull request by stbenjam**
- Add note to main page
- Add `site.latest` for links to latest version
- Add "Stable" as first menu option

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,6 +65,9 @@ <h1 class="tagline col-md-5">
       </div>
       <div class="col-md-3 col-md-offset-1 news">
         <p class="snippet">
+        Katello 2.1 is officially released! Click the 'Get Started' button above to install yours today! Also <a href="/docs/2.1/release_notes/release_notes.html">read the release notes</a> or <a href="/docs/2.1/upgrade/index.html">upgrade instructions.</a>
```
Comment (ehelms) [inline index.html]: Nitpick - indent

File: index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h1 class="tagline col-md-5">
             <i class="fa fa-plus"></i>
             <img src="{{ site.baseurl }}/assets/images/foreman.png" alt="Foreman"/>
           </p>
-          <a href="{{ site.baseurl }}/docs/2.0/installation/index.html" class="get-started">
+          <a href="{{ site.baseurl }}/docs/{{ site.latest }}/installation/index.html" class="get-started">
```
Comment (jlsherrill) [inline index.html]: I would just hardcode this to '2.1'.  Mainly so we can add 2.2 docs without users clicking 'get started' and being sent to 2.2

Comment (stbenjam) [inline index.html]: @jlsherrill `latest` is set in _config.yml, so you would only update that on GA announcement (like here).  You can add KATELLO-2.2 branch w/out affecting this.

Comment (stbenjam) [inline index.html]: The other benefit here is from unversioned parts of the site can link to "/docs/{{ site.latest }}/whatever" now so you don't link users to nightly, but to latest stable.  The idea came from Foreman's site, which does something similar.

Comment (jlsherrill) [inline index.html]: ahhh, i'm good with that

**Comment by jlsherrill** [comment]
ACK

**Comment by stbenjam** [comment]
Woooo!

**Pull request merged**